libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-wrapper.component.ts

```diff
@@ -87,6 +99,28 @@ export class SkyAgGridWrapperComponent
     }
     this.agGrid.gridReady.pipe(takeUntil(this.#ngUnsubscribe)).subscribe(() => {
       this.#moveHorizontalScroll();
+      this.agGrid.columnApi.getAllDisplayedColumns().forEach((column) => {
+        this.#updateInlineHelp(column);
+      });
+
+      [
+        Events.EVENT_COLUMN_MOVED,
+        Events.EVENT_COLUMN_RESIZED,
+        Events.EVENT_COLUMN_VISIBLE,
+        Events.EVENT_DISPLAYED_COLUMNS_CHANGED,
+        Events.EVENT_GRID_COLUMNS_CHANGED,
+        Events.EVENT_NEW_COLUMNS_LOADED,
+        Events.EVENT_VIEWPORT_CHANGED,
+        Events.EVENT_VIRTUAL_COLUMNS_CHANGED,
+      ].forEach((event) => {
+        fromEvent(this.agGrid.api, event)
+          .pipe(takeUntil(this.#ngUnsubscribe), debounceTime(10))
+          .subscribe(() => {
+            this.agGrid.columnApi.getAllDisplayedColumns().forEach((column) => {
+              this.#updateInlineHelp(column);
+            });
+          });
+      });
```

Comment:
Watch for changes and re-check each column. AG Grid aggressively removes offscreen elements for its virtual DOM as a way to mitigate large datasets. It was necessary to subscribe to this set of events to catch all of the various cases. I also looked into using a mutation observer but that was uglier.